kuzzleio/sdk-c
# Export C SDK alongside cgo source files

## What does this PR do ?

Contains the C SDK's alongside cgo source files.

### How should this be manually tested?

```sh
git clone --recursive git@github.com:kuzzleio/sdk-c.git
cd sdk-c
git submodule init && git submodule update
git checkout KZL-490-c-from-go
cd go/src/github.com/kuzzleio/sdk-go && git checkout 1-dev && cd -
make
ls build/
```
You should find those files:
- kuzzle.h
- libkuzzlesdk.a
- libkuzzlesdk.a.1.0.0
- libkuzzlesdk.so
- libkuzzlesdk.so.1.0.0
